tests/callbacks/test_stochastic_weight_avg.py

```diff
@@ -139,22 +145,22 @@ def train_with_swa(
 
 @RunIf(min_gpus=2, standalone=True)
 def test_swa_callback_ddp(tmpdir):
-    train_with_swa(tmpdir, strategy="ddp", gpus=2)
+    train_with_swa(tmpdir, strategy="ddp", accelerator="gpu", devices=2)
 
 
 @RunIf(min_gpus=2)
 def test_swa_callback_ddp_spawn(tmpdir):
-    train_with_swa(tmpdir, strategy="ddp_spawn", gpus=2)
+    train_with_swa(tmpdir, strategy="ddp_spawn", accelerator="gpu", devices=2)
 
 
 @RunIf(skip_windows=True, skip_49370=True)
 def test_swa_callback_ddp_cpu(tmpdir):
-    train_with_swa(tmpdir, strategy="ddp_spawn", num_processes=2)
```

Comment:
Thanks for catching that! I'll use `grep` next time to make sure I catch them all. Using the regular "Find" doesn't do the trick.

Reply:
Actually, the accelerator is already set to use "cpu". But I'll make that more explicit by adding `accelerator="cpu"`

Reply:
It use cpu because the test machine (your laptop?) only have cpu, if this test run in ci and has GPUs or TPUs it will choose these over CPU. I think so, we need specific accelerator="cpu"

Reply:
The function is already set to use cpu, see [this line](https://github.com/mathemusician/pytorch-lightning/blob/944d6772f28bbc1c4ea8ce38431e95de04faafef/tests/callbacks/test_stochastic_weight_avg.py#L114)